testing/dbGenTests.test.js

```diff
@@ -1,116 +0,0 @@
-const generator = require('../db/dbGeneration');
```

Comment:
Just wondering what happened to this file. It looks like it was placed into a new file with the same name? Was there an issue with this test?

Reply:
i'm not sure I get the question. 
This line is importing all the dbGeneration functions from dbGeneration.js into the test file

Reply:
Oh it looks like according to the git file changes, the git this entire file was deleted and replaced with a similar file. Maybe its just some strange git issue. My comment is not related to line 1, but the entire dbGenTest.test.js in general.